src/Pyz/Yves/CartPage/Theme/default/components/molecules/product-cart-item/product-cart-item.twig

```diff
@@ -0,0 +1,152 @@
+                                {% endif %}
+                            {% endblock %}
+
+                            {% block quantity %}
+                                {% set isEmptyProductInfo = amountSalesUnit is empty and quantitySalesUnit is empty and data.quantity is empty %}
+
+                                {% if isEmptyProductInfo %}
+                                    <div class="{{ config.name }}__quantity-holder">
+                                        {% include molecule('product-item-quantity') with {
+                                            data: {
+                                                productItem: data.listItem,
+                                                isDisabled: not data.canEdit,
+                                            },
+                                        } only %}
+                                    </div>
+                                {% endif %}
+                            {% endblock %}
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="{{ config.name }}__inner grid grid--stretch grid--gap">
+            <div class="{{ config.name }}__col {{ config.name }}__col--content col">
+                {% block note %}
+                    {% widget 'CartItemNoteFormWidget' args [data.listItem, data.list] with {
+                        data: {
+                            isTitleVisible: false,
+                        },
+                    } only %}
+                    {% endwidget %}
+                {% endblock %}
+
+                {% if data.listItem.productOptions is defined and data.listItem.productOptions %}
+                    {% include molecule('product-item-options') with {
+                        modifiers: modifiers,
+                        data: {
+                            productItem: data.listItem,
+                        },
+                    } only %}
+                {% endif %}
+
+                {% if data.options %}
+                    {% include molecule('product-item-options') with {
+                        data: {
+                            options: data.options,
+                        },
+                    } only %}
+                {% endif %}
+            </div>
+
+            <div class="{{ config.name }}__col {{ config.name }}__col--total col">
+                <div class="{{ config.name }}__inner {{ config.name }}__inner--actions {{ config.name }}__inner--full-height grid grid--column grid--justify-column">
+                    <div class="col col--sm-12">
+                        <div class="{{ config.name }}__inner {{ config.name }}__inner--total grid grid--column">
+                            {% block totalPrice %}
+                                {% if can('SeePricePermissionPlugin') %}
+                                    {% set totalPrice = data.listItem.unitPrice is defined ? data.listItem.sumSubtotalAggregation : data.listItem.price %}
```

Comment:
Please add a separate variable for 'data.listItem' to avoid repeating requests to the data